src/Jackalope/ObjectManager.php

```diff
@@ -1806,4 +1806,22 @@ public function purgeDisappearedNode($absPath, $keepChanges)
         // objectsByPath and the calling parent node can forget it
         return true;
     }
+
+    /**
+     * Updates a single node via the transport layer if the node is modified
+     * @param NodeInterface $node The node to update
+     */
+    private function updateNode($node)
+    {
+        /** @var $node Node */
```

Comment:
I was not sure if you rely on this exception being thrown somewhere, that's why I just extracted it. Will change it that way.